Automattic/woocommerce-payments
# Use base64 image for the menu icon

Fixes #3686 

#### Changes proposed in this Pull Request

This PR uses the base64 icon for the menu.

https://github.com/Automattic/wc-calypso-bridge/pull/777 must be deployed first before releasing the change in this PR. Otherwise, the menu icon might be rendered as a blank space on some pages on wp.com sites.

#### Testing instructions

1. Clone this branch.
2. Make sure the menu icon is still rendered correctly.
3. Inspect the menu icon and confirm the menu is using the base64 image data.

![Screen Shot 2022-01-27 at 2 45 28 PM](https://user-images.githubusercontent.com/4723145/151455826-78b9c10e-f3d0-44bb-95b7-d4207f0d4eb3.jpg)
